test/jdk/java/util/Formatter/Basic.java

```diff
@@ -101,6 +90,6 @@ public static void main(String[] args) {
             throw new RuntimeException((fail + pass) + " tests: "
                                        + fail + " failure(s), first", first);
         else
-            out.println("all " + (fail + pass) + " tests passed");
+            System.out.printf("all " + pass + " tests passed");
```

Comment:
This line is missing a newline; add a \n, or use println().

Reply:
I dropped the new line since when the output comes out of the sub process log

Without the newline it looks like 
[ ... ] 

vs

[...
]